### ISSUE TITLE
firmware 

### ISSUE THREAD

**Issue by TREBOR81400** [wuxx/AntRunner#10]
Good morning,
Can I directly flash the firmware file “flash_image_20220821.bin” onto the universal controller card “6 Pack Universal CNC Controller by Bart Dring of Tindie” to operate an antenna rotator? Will this work directly without modification? THANKS

**Comment by wuxx** [comment]
Hello, it probably won’t work. You can refer to the schematic for details.

**Comment by TREBOR81400** [comment]
hello, thank you for your response, but I have already, through clumsiness, broken down a card that you kindly resold to me, telling me that it was a small series. I actually wish, as the Tindie site does not offer not a codified replacement card, have an "other" technical possibility to be able to continue using my AntRunner in the event of a new breakdown, particularly on the USB connection which I find fragile. Hence my search for a possible alternative solution. THANKS

    Le vendredi 20 octobre 2023 à 05:27:24 UTC+2, wuxx ***@***.***> a écrit :  
 
 


Hello, it probably won’t work. You can refer to the schematic for details.

**Comment by wuxx** [comment]
Hello, is the TYPE-C USB interface damaged? You can use an external USB to UART tool to access it.

**Comment by wuxx** [comment]
![image](https://github.com/wuxx/AntRunner/assets/1648489/ba727bf4-34ec-464e-83e8-189456b16fec)

**Comment by wuxx** [comment]
![image](https://github.com/wuxx/AntRunner/assets/1648489/d18536dd-e846-4acc-9b1a-0c3de8a52e73)

**Comment by wuxx** [comment]
Unplug the J5 short-circuit cap and connect it with an external USB to UART tool.

**Comment by TREBOR81400** [comment]
thank you but it's too thin to solder for me, it's not easy to find a connector, I would prefer that you put a whole cart for sale as a reference and I will order it from you, ceal will make me a backup card just in case, THANKS

    Le vendredi 20 octobre 2023 à 13:37:12 UTC+2, wuxx ***@***.***> a écrit :  
 
 


Unplug the J5 short-circuit cap and connect it with an external USB to UART tool.

**Comment by wuxx** [comment]
Okay, please contact us through the platform you purchased

**Comment by TREBOR81400** [comment]
thank you, but I don't see the complete electronic card on your Tindie store alone

    Le dimanche 22 octobre 2023 à 02:39:34 UTC+2, wuxx ***@***.***> a écrit :  
 
 


Okay, please contact us through the platform you purchased

**Comment by wuxx** [comment]
Yes, individual boards cannot work independently, so there is currently no separate link

**Comment by TREBOR81400** [comment]
so I was stupid to buy an AntRunner when I can't order spare parts in case of breakdown

    Le lundi 23 octobre 2023 à 03:07:38 UTC+2, wuxx ***@***.***> a écrit :  
 
 


Yes, individual boards cannot work independently, so there is currently no separate link

**Comment by wuxx** [comment]
It’s no problem to purchase the parts separately, just contact us please.

**Comment by TREBOR81400** [comment]
Merci
    Le mardi 24 octobre 2023 à 02:15:34 UTC+2, wuxx ***@***.***> a écrit :  
 
 


It’s no problem to purchase the parts separately, just contact us please.
